sql/core/src/test/resources/sql-tests/results/ansi/array.sql.out

```diff
@@ -128,7 +128,7 @@ select sort_array(array('b', 'd'), '1')
 struct<>
 -- !query output
 org.apache.spark.sql.AnalysisException
-cannot resolve 'sort_array(array('b', 'd'), '1')' due to data type mismatch: Sort order in second argument requires a boolean literal.; line 1 pos 7
+{"errorClass":null,"messageParameters":[],"queryContext":[]}
```

Comment:
I'm with @entong, we can't do this.
If there is no defined error class we need have a default.
How about:
{"errorClass":"legacy","messageParameters":["message" -> "<original message>"],"queryContext":[]}

Reply:
Hm, it's a bit odd to me that the user-facing error contains JSON exception. e.g., if you run it in `spark-sql` then users would face such JSON encoded string .. that I don't think it's good.

Reply:
@HyukjinKwon This is not user-facing changes. I changed the way how golden files are generated in tests. Output of `spark-sql` is the same.

Reply:
> How about:
> {"errorClass":"legacy","messageParameters":["message" -> "original message"],"queryContext":[]}

It makes sense. Let me do that. Thanks.